kpops/cli/main.py

```diff
@@ -191,7 +192,9 @@ def run_destroy_clean_reset(
     )
     for component in reversed(get_steps_to_apply(pipeline, steps)):
         log_action("Destroy", component)
-        component.destroy(dry_run=dry_run, clean=clean, delete_outputs=delete_outputs)
+        component.destroy(dry_run=dry_run)
+        if clean and isinstance(component, KafkaApp):
```

Comment:
Why do you check for is instance? I would just make the default implementation do nothing and so far only KafkaApp overrides it

Reply:
haha, you can read my mind 😂